### ISSUE TITLE
Apple silicon - triggers has same errro as for pipelines: Error executing command: fork/exec

### ISSUE THREAD

**Issue by JCzz** [tektoncd/triggers#1233]
Hi

I am looking at using triggers, and have run into an error similar to one i have seen in Pipelines - where I got a workaround, see this: https://github.com/tektoncd/pipeline/issues/4247

I triggers I get the following error message: `Error executing command: fork/exec /bin/sh: exec format error`

Now in triggers I dont know how to work around it the same way. I can see the trigger controller contains some more specific images for triggers controller & eventlistenersink.

In short and in pipeline it worked by adding this to pipeline-controller:
```
- shell-image
- busybox@sha256:149ff441a10b7e05b3c60da0916f98405e9bb551923fefd930079d1a55c01ce0
```

Do you know how to fix/work around this in triggers - until there is a fix?

Thanks in advance

**Comment by dibyom** [comment]
Is the error from the Triggers controller or an EventListener? Having the exact error would be helpful!

Guessing here: the one place where we do declare a base image is https://github.com/tektoncd/triggers/blob/main/.ko.yaml#L1

Maybe you can try switching that to the image that worked for you? But in that case, you will have to build and install [from source](https://github.com/tektoncd/triggers/blob/main/DEVELOPMENT.md#install-triggers)

**Comment by JCzz** [comment]
Thanks @dibyom and good point regarding error message :-)

The message is: `Error executing command: fork/exec /bin/sh: exec format error`

Sounds like a good solution, but I think I will refrain from building.

Do you know if it possible to call/start pipelines from other solutions. I am using Knative for deployment and maybe I can use Knative Eventing?

**Comment by JCzz** [comment]
Over at Slack I had a chat with Andrea Frittoli. I asked how I could use Knative Eventing and he suggested curl, kubectl, tkn or golang client. I have decided to move in that direction until eventually Tekton trigger, and pipelines for that matter, is working out of the box on arm64 ;-)

I will keep this issue open for arm64 to come - ok?
